go/tasks/plugins/array/catalog.go

```diff
@@ -38,7 +38,7 @@ func DetermineDiscoverability(ctx context.Context, tCtx core.TaskExecutionContex
 	}
 
 	// Extract the custom plugin pb
-	arrayJob, err := arrayCore.ToArrayJob(taskTemplate.GetCustom(), taskTemplate.TaskTypeVersion)
+	arrayJob, err := arrayCore.ToArrayJob(taskTemplate.GetConfig(), taskTemplate.TaskTypeVersion)
```

Comment:
You will have to leave code in here to keep backward compatibility. Something like:
* check for config... if there, use that
* If not, fallback to GetCustom